### ISSUE TITLE
Brave Browser Problem

### ISSUE THREAD

**Issue by AdoNikeOFFICE** [vaadin/flow#15105]
Hello,

it seems like Vaadin does not work with the Brave browser.
![image](https://user-images.githubusercontent.com/65062119/201481407-97000774-97ac-42d3-ba9a-6efe837dff38.png)

Error from the image
```
TypeError: Failed to fetch dynamically imported module: http://localhost:8080/VAADIN/@fs/C:/Users/adoni/IdeaProjects/my-app/target/frontend/generated-flow-imports.js
```

If I try Edge, for example, it works like a charm.
![image](https://user-images.githubusercontent.com/65062119/201481501-1b3398d5-0957-4514-999f-c733d0c94d93.png)

**Comment by Artur-** [comment]
Is there something more about the reason in the browser console?

**Comment by AdoNikeOFFICE** [comment]
> Is there something more about the reason in the browser console?

I didn't notice anything.

**Comment by Artur-** [comment]
```
npx @vaadin/cli init hello-brave
cd hello-brave
mvn
```

works fine in Brave on mac os

**Comment by knoobie** [comment]
Just a stupid guess, probably related to Brave on Windows and some security stuff because `/@fs/C:/Users` could potentielle look like a path traversal attack.. I've seen other issues with Brave in the past as well, for example in the auth2 example project https://github.com/vaadin/vaadin-oauth-example/issues/43

**Comment by AdoNikeOFFICE** [comment]
Yes, it's issue related to Windows, I don't know haha, but it just doesn't work..

**Comment by caalador** [comment]
Which version of Brave + Flow + Window does this happen with?
I've been basically using Brave for the past 3 years for Flow development and testing and have not had this problem.
I'm running `[Version 1.45.123 Chromium: 107.0.5304.110 (Official Build) (64-bit)]`

**Comment by AdoNikeOFFICE** [comment]
> Which version of Brave + Flow + Window does this happen with? I've been basically using Brave for the past 3 years for Flow development and testing and have not had this problem. I'm running `[Version 1.45.123 Chromium: 107.0.5304.110 (Official Build) (64-bit)]` with Win11

I downloaded newest available Brave Chromium 107.1.45.123 + Win11

**Comment by caalador** [comment]
I'm getting this now also when I made a new 23.2.8 spring-boot project, but not for a project with 23.2.8 using jetty.

**Comment by knoobie** [comment]
Another user reports the same problem: https://discord.com/channels/732335336448852018/1043624798074441881/1043631590682664960

**Comment by seso975** [comment]
Hi. I am trying to migrate 14.2.x to 23.2.x.
In dev mode View is displayed, but without flow comps (attached logs). Production mode is ok.
config: v 23.2.10, java 11, Vaadin CDI 

I tried on Win10,  firefox 107.0.1, Edge 107.0.1418.62, chrome 108.0.5359.71 . All broken.
[localhost-1669902412398.log](https://github.com/vaadin/flow/files/10139499/localhost-1669902412398.log)
![v-console](https://user-images.githubusercontent.com/17812213/205252980-83acba46-a016-465a-bb7d-674ea29edeba.png)
![v-console2](https://user-images.githubusercontent.com/17812213/205252984-6ec82866-66ff-471d-a072-c6eccd53ddee.png)

**Comment by knoobie** [comment]
There was another user report about it in discord and a workaround - disabling Brave Shield for localhost worked. https://discord.com/channels/732335336448852018/935356359954812988/1063497344068366368

**Comment by letrthang** [comment]
It also happens on 
Vaadin 23.3. JDK 17. MacOS.

https://discord.com/channels/732335336448852018/1119550672812130324

**Comment by letrthang** [comment]
so this is issue on Vaadin or Brave ?

**Comment by caalador** [comment]
The issue would seem to be the brave browser shield blocking the `@fs` requests as disabling the shield fixes the issue.
In V24 the issue is not present when using the devBundle feature as there are no `@fs` requests to the server.

**Comment by Artur-** [comment]
You should create an issue for Brave if it is preventing loading of URLs used by Vite. There is nothing wrong with the URLs, they just happen to be written in a certain way

**Comment by AbdullahGheith** [comment]
I just started using brave on windows and suddenly my application doesnt work anymore.
